### PR TITLE
fix:#570 used the Response class  and set the input for for status_code to 200

### PR DESCRIPTION
**Pull request by ayushjha119**
@sabaimran 
I have used the Response class  and set the input for for status_code to 200 and also remove the comment.

### CODE REVIEW

**Review by sabaimran — APPROVED** [review]
Awesome! Thanks for addressing those comments @ayushjha119 .

**Comment by sabaimran** [comment]
Closes #570

**Comment by ayushjha119** [comment]
@sabaimran I am new to open source, it is showing like this ..i am attaching the screenshot and issue is still appearing in issue seaction
have i done anything wrong here?? 

Screenshot 2023-12-21 172102

File: src/khoj/routers/api.py [hunk 1]
```diff
@@ -862,3 +862,7 @@ async def extract_references_and_questions(
         compiled_references = [item.additional["compiled"] for item in result_list]
 
     return compiled_references, inferred_queries, defiltered_query
+
+@app.get("/health")
```
Comment (sabaimran) [inline src/khoj/routers/api.py:866]: Ah, my bad for not realizing this before, but this should be as such
```suggestion
@api.get("/health")
```

You can run the unit tests locally by enabling your virtual environment with the development packages and then running `pytest` from the cli.

**Comment by sabaimran** [comment]
Closing as new PR raised in #592 to address changes requested

**Pull request closed without merging**